sites/kit.svelte.dev/src/lib/docs/server/index.js

```diff
@@ -120,7 +120,9 @@ export async function read_file(file) {
 				version_class = ' js-version';
 			}
 
-			if (language === 'js' || language === 'ts') {
+			if (language === 'ts') {
+				html = renderCodeToHTML(source, 'ts', { twoslash: false }, {}, highlighter);
```

Comment:
What's the reason for this change? Doesn't this remove these hover hints for all TS blocks (which also affects all other doc pages when people toggle the typescript version of code snippets)?

Reply:
Blurgh yes it does — we need to distinguish between (effectively) `.ts` and `.d.ts`. On the `.d.ts` blocks they're noisy and useless